plant/notifications/api/views.py

```diff
@@ -0,0 +1,21 @@
+from django.shortcuts import render
+from rest_framework.mixins import ListModelMixin
+from rest_framework.viewsets import GenericViewSet
+
+from plant.notifications.api.serializers import NotificationSerializer
+from plant.notifications.models import Notification
+
+
+# Create your views here.
```

Comment:
I prefer to remove comments like this one in L9 as it's pre generated and does not have any value